example/.flutter-plugins-dependencies

```diff
@@ -1 +1 @@
-{"_info":"// This is a generated file; do not edit or check into version control.","dependencyGraph":[{"name":"audioplayers","dependencies":["path_provider"]},{"name":"path_provider","dependencies":[]}]}
\ No newline at end of file
+{"info":"This is a generated file; do not edit or check into version control.","plugins":{"ios":[{"name":"audioplayers","path":"/home/erick/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.13.7/","dependencies":["path_provider"]},{"name":"path_provider","path":"/home/erick/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.5.1/","dependencies":[]}],"android":[{"name":"audioplayers","path":"/home/erick/flutter/.pub-cache/hosted/pub.dartlang.org/audioplayers-0.13.7/","dependencies":["path_provider"]},{"name":"path_provider","path":"/home/erick/flutter/.pub-cache/hosted/pub.dartlang.org/path_provider-1.5.1/","dependencies":[]}],"macos":[],"linux":[],"windows":[],"web":[]},"dependencyGraph":[{"name":"audioplayers","dependencies":["path_provider"]},{"name":"path_provider","dependencies":[]}],"date_created":"2020-01-26 10:58:34.863327","version":"1.14.5-pre.37"}
```

Comment:
`do not edit or check into version control`

please add to gitignore and remove it